### PR TITLE
fix #273032: implement packages to install soundfont, templates, etc.

### PR DESCRIPTION
**Pull request by lasconic**

### CODE REVIEW

**Comment by lasconic** [comment]
Also include https://github.com/musescore/MuseScore/pull/3739

**Comment by lasconic** [comment]
Also include https://github.com/musescore/MuseScore/pull/3743/ for master

**Comment by lasconic** [comment]
https://github.com/musescore/MuseScore/pull/3747

**Comment by lasconic** [comment]
Added https://github.com/musescore/MuseScore/pull/3754

**Comment by lasconic** [comment]
Done.

**Comment by Jojo-Schmitz** [comment]
And now #3761 ;-)

**Comment by lasconic** [comment]
Done

**Comment by Jojo-Schmitz** [comment]
and now #3773 :-)

**Comment by anatoly-os** [comment]
See https://github.com/musescore/MuseScore/commit/d21bda33ce8872b559c0a8c76acc87b31fe947f6

**Comment by anatoly-os** [comment]
@lasconic f4148d874d84e24b81f2cbebdcfcd90c9b10281e, d402925f928d33af776eadc69009375d9e4f5cc0 are missed. They could be squashed btw.

File: mscore/resourceManager.cpp [hunk 1]
```diff
@@ -196,6 +309,79 @@ void ResourceManager::download()
...
+      {
+      QPushButton* button = static_cast<QPushButton*>(sender());
+      QString data = button->property("path").toString();
+      QString hash = button->property("hash").toString();
+      button->setText(tr("Updating"));
+      button->setDisabled(true);
+      QString baseAddress = baseAddr() + data;
+      DownloadUtils dl(this);
+      dl.setTarget(baseAddress);
+      QString localPath = QDir::tempPath() + QDir::separator() + data.split('/')[1];
```
Comment (Jojo-Schmitz) [inline mscore/resourceManager.cpp]: See #3785 and the chamhe from `QDir::separator()` to `"/"`

**Pull request merged**